### ISSUE TITLE
How to customize legend on the plot using the WISP wrapper

### ISSUE THREAD

**Issue by davidshen84** [andypetrella/spark-notebook#339]
Fist, does the wrapper supports customizing the legend on the plot? If not, maybe we can add it? I could not find a reference on the wiki page.

**Comment by andypetrella** [comment]
Yeah, the best would then be to use the wrapper that encapsulate the Highchart object directly, so you have the full control → see https://github.com/andypetrella/spark-notebook/blob/master/modules/common/src/main/scala/notebook/front/third/WISP.scala#L65

How to use it then would be more a wisp documentation, but I can still try to help you afterwards
